playbook/app/pb_kits/playbook/pb_table/table_header.html.erb

```diff
@@ -2,15 +2,14 @@
     colspan: object.colspan,
     aria: object.aria,
     class: object.classname,
-    data: object.data,
-    id: "pb-th#{object.id}") do %>
+    data: object.data ) do %>
   <% unless sorting_style? %>
-    <%= pb_rails("flex", props:{ align: object.align_content, justify: object.justify_sort_icon, classname: "pb_th_nolink" }) do %>
+    <%= pb_rails("flex", props:{ align: object.align_content, justify: object.justify_sort_icon, classname: "pb_th_nolink", id: "pb-th#{object.id}" }) do %>
```

Comment:
Did we alpha test this? It's technically a possiblity that this could be a breaking change.

We're moving the id from one spot on an existing kit, to a different spot. If users are already styling things using an `id` for example in a specific spot in nitro, this could break their things. 

Just trying to make sure we understand that, and have tested it.